src/services/notification.ts

```diff
@@ -12,7 +12,7 @@ const NotificationApi = createApi({
       string | null | undefined
     >({
       query: walletAddress => ({
-        url: `${KS_SETTING_API}/v1/topic-groups`,
+        url: `${'https://ks-setting.stg.kyberengineering.io/api'}/v1/topic-groups`,
```

Comment:
revert

Reply:
when testing finished, u can revert, add todo

Reply:
remove this file.